Add file size to json object && some tests

SimplyAhmazing opened the pull request:
Added a property that has a file/directory's size in bytes. I needed this for a project I'm working on, maybe others will want it too. Added tests.

mihneadb commented:
Cool, thanks! Care to also set up a travis yml file for running those tests all the time? :)

SimplyAhmazing commented:
Hey! I set up a travis yml file (first time I've done one of those) and I had to rejigger the tests. I also discovered and fixed a bug.

The `extensions` parameter didn't ignore 'dot files' so I added a part to the code that did that.

mihneadb commented:
Sweet, thanks!

mihneadb commented on README.md:
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/SimplyAhmazing/node-directory-tree.svg)](https://travis-ci.org/SimplyAhmazing/node-directory-tree)
```
Please update this to the original URL of the repo :D.

mihneadb commented:
Good to merge after that URL fix :). Thanks!

SimplyAhmazing commented:
I fixed the url!

mihneadb commented:
Sweet, thanks!
